sdks/python/apache_beam/internal/gcp/json_value.py

```diff
@@ -53,7 +53,7 @@ def get_typed_value_descriptor(obj):
     ~exceptions.TypeError: if the Python object has a type that is not
       supported.
   """
-  if isinstance(obj, (str, unicode)):
+  if isinstance(obj, (bytes, unicode)):
```

Comment:
Thanks! Let's update docstrings for get_typed_value_descriptor and to_json_value.